sdk/identity/Azure.Identity/samples/OtherCredentialSamples.md

```diff
@@ -11,9 +11,11 @@ To use `AzurePipelinesCredential`, configure the following values in the constru
 
 1. `clientId`: Client ID from your user-assigned managed identity OR Application (client) ID from your app registration.
 2. `tenantId`: Tenant ID from your user-assigned managed identity OR Directory (tenant) ID from your app registration.
-3. `serviceConnectionId`: The service connection ID is the **GUID representing your service connection**. The value is obtained by looking at the browser's address bar when you navigate to a service connection in the Azure Pipelines. It's the `resourceId` value, as found in the URL's query string.
-   ![resourceId value, as found in the query string of the Azure Resource Manager service connection created in Azure Pipelines](exampleServiceConnectionUrl.png)
-4. `systemAccessToken`: [See how to configure the predefined system variable System.AccessToken for the Azure Pipelines task](https://learn.microsoft.com/azure/devops/pipelines/build/variables?view=azure-devops&tabs=yaml#systemaccesstoken). This is the value you'll pass to the credential's constructor.
+3. `serviceConnectionId`: The service connection ID is the **GUID representing your service connection**. Once you navigate to an Azure Pipelines service connection details page, the value is obtained in one of the following ways:
+    1. Copy the **ID:** value that appears below the service connection name.
```

Comment:
The annotation difference here is unusual.   Bolded "ID" and a code block for "resourceId" - was that intentional?

Reply:
It was intentional. The thinking is this content will soon move to Learn. On Learn, bolded text signifies a UI element. Code-fencing is used for anything that shouldn't be machine-translated.